kafka/util.go

```diff
@@ -102,20 +102,7 @@ func parseProducerQuery(query url.Values) *sarama.Config {
 
 func parseSubscriberQuery(query url.Values) *sarama.Config {
```

Comment:
this method still needed?

Reply:
Only as a placeholder for future. No harm in keeping it, I think.